python_modules/libraries/dagstermill/dagstermill/io_managers.py

```diff
@@ -1,22 +1,22 @@
 import os
 from pathlib import Path
-from typing import Any, Optional, Sequence
+from typing import Any, List, Optional
 
 import dagster._check as check
-from dagster import AssetKey, AssetMaterialization
-from dagster._config import Field
+from dagster import AssetKey, AssetMaterialization, ConfigurableIOManager
+from dagster._config.structured_config import infer_schema_from_config_class
 from dagster._core.definitions.metadata import MetadataValue
 from dagster._core.execution.context.input import InputContext
 from dagster._core.execution.context.output import OutputContext
-from dagster._core.storage.io_manager import IOManager, io_manager
+from dagster._core.storage.io_manager import io_manager
 from dagster._utils import mkdir_p
+from pydantic import Field as PyField
 
 from dagstermill.factory import _clean_path_for_windows
 
 
-class OutputNotebookIOManager(IOManager):
-    def __init__(self, asset_key_prefix: Optional[Sequence[str]] = None):
-        self.asset_key_prefix = asset_key_prefix if asset_key_prefix else []
+class OutputNotebookIOManager(ConfigurableIOManager):
```

Comment:
hmmm since `LocalOutputNotebookIOManager` implements `create_io_manager` should this inherit from `ConfigurableIOManagerFactory`